client/web/components/query-renderer/index.js

```diff
@@ -1,21 +1,29 @@
-import React from 'react'
-import { QueryRenderer } from 'react-relay'
-import environment from '../../util/environment'
+// @flow
 
-export default function AppQueryRenderer(props) {
+import React from 'react';
+import { QueryRenderer } from 'react-relay';
+import environment from '../../util/environment';
+
+type Props = {
+  query: string,
+  render: (rendererProps: {}) => mixed,
+};
```

Comment:
You could even improve this type to be *exact* and *read-only* like this:

```
type Props = {|
  +query: string,
  +render: (rendererProps: {}) => mixed,
|};
```

Also, you should use the `rendererProps` type from the `__generated__` file.